test/Test/Main.purs

```diff
@@ -155,5 +155,10 @@ testRecordInstances = do
 
 testSignum :: AlmostEff
 testSignum = do
-  assert "signum positive zero" $ signum 0.0 == 0.0
-  assert "signum negative zero" $ signum (-0.0) == (-0.0)
+  assert "signum positive zero" $ objectIs (signum 0.0) 0.0
+  assert "signum negative zero" $ objectIs (signum (-0.0)) (-0.0)
+
+-- Seems to be only way to check whether zero is `-0.0` or `0.0`
```

Comment:
I forgot that our `show` doesn't distinguish negative and positive zero. But, you can still distinguish them without the FFI:
```
show (1.0/0.0) == "Infinity"
show (1.0/(-0.0)) == "-Infinity"
```